crates/bevy_remote/src/lib.rs

```diff
@@ -136,6 +136,7 @@
 //!   - `components` (optional): An array of [fully-qualified type names] of components to fetch,
 //!     see _below_ example for a query to list all the type names in **your** project.
 //!   - `option` (optional): An array of fully-qualified type names of components to fetch optionally.
+//!     to fetch all reflectable components, you can pass in the string `"All"`.
```

Comment:
```suggestion
//!     to fetch all reflectable components, you can pass in the string `"all"`.
```
lowercase would be more expected imo